lib/src/request/ResourceTiming.ts

```diff
@@ -16,8 +16,5 @@ export type ResourceTiming = {
 	responseStart?: number
 
 	// The timestamp immediately after the browser receives the last byte of the resource or immediately before the transport connection is closed, whichever comes first.
-	responseEnd?: number
-
-	// The difference between the responseEnd and the startTime properties.
-	duration?: number
+	responseEnd: number
```

Comment:
This should be reversed: `duration` should be required, and `responseEnd` removed (for now).

Reply:
No problems, I'll update it later today

Reply:
Addressed it in https://github.com/streaming-video-technology-alliance/common-media-library/pull/170/commits/779d5f6f87d98891038d71e2c5dbe62cddb01d35